src/Prima/Pyxis/Form.elm

```diff
@@ -1579,51 +1639,249 @@ validate model config validation =
+        ( Expression Error exp _, FormFieldPasswordConfig { reader } _ ) ->
             (Regex.contains exp << Maybe.withDefault "" << reader) model
 
-        ( Expression exp _, FormFieldTextareaConfig { reader } _ ) ->
+        ( Expression Error exp _, FormFieldTextareaConfig { reader } _ ) ->
             (Regex.contains exp << Maybe.withDefault "" << reader) model
 
-        ( Expression exp _, FormFieldAutocompleteConfig { choiceReader } _ ) ->
+        ( Expression Error exp _, FormFieldAutocompleteConfig { choiceReader } _ ) ->
             (Regex.contains exp << Maybe.withDefault "" << choiceReader) model
 
-        ( Expression exp _, _ ) ->
+        ( Expression Error exp _, _ ) ->
             True
 
-        ( Custom validator _, _ ) ->
+        ( Custom Error validator _, _ ) ->
             validator model
 
         ( _, FormFieldPureHtmlConfig _ ) ->
             True
 
+        ( NotEmpty Warning _, FormFieldAutocompleteConfig _ _ ) ->
+            True
+
+        ( NotEmpty Warning _, FormFieldCheckboxConfig _ _ ) ->
+            True
+
+        ( NotEmpty Warning _, FormFieldDatepickerConfig _ _ ) ->
+            True
+
+        ( NotEmpty Warning _, FormFieldPasswordConfig _ _ ) ->
+            True
+
+        ( NotEmpty Warning _, FormFieldRadioConfig _ _ ) ->
+            True
+
+        ( NotEmpty Warning _, FormFieldSelectConfig _ _ ) ->
+            True
+
+        ( NotEmpty Warning _, FormFieldTextareaConfig _ _ ) ->
+            True
+
+        ( NotEmpty Warning _, FormFieldTextConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldAutocompleteConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldCheckboxConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldDatepickerConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldPasswordConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldRadioConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldSelectConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldTextareaConfig _ _ ) ->
+            True
+
+        ( Expression Warning _ _, FormFieldTextConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldAutocompleteConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldCheckboxConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldDatepickerConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldPasswordConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldRadioConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldSelectConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldTextareaConfig _ _ ) ->
+            True
+
+        ( Custom Warning _ _, FormFieldTextConfig _ _ ) ->
+            True
```

Comment:
ma servono tutti sti branch? Tanto sono tutti True, puoi anche fare
```
_ ->
  True
```

ovvio il rischio è quello di aggiungere un caso in cui sarebbe da considerare False, e il compilatore funzionerebbe lo stesso...che dici @alessiosantangelo?